src/_components/shared/highlight_banner.erb

```diff
@@ -0,0 +1,17 @@
+<highlight-banner>
+  <div class="wrapper">
+    <div class="content">
+      <h3><%= @title %></h3>
+      <%= content %>
+      <% if @link_url.present? && has_button_content? %>
```

Comment:
what about including `@link_url.present?` in `has_button_content?`

Reply:
✅